modules/oe_content_sub_entity/modules/oe_content_sub_entity_author/tests/src/Kernel/AuthorSkosUpdaterTest.php

```diff
@@ -80,6 +80,11 @@ protected function setUp(): void {
    * Tests update of author values to sub-entities by the service.
    */
   public function testNodeUpdate(): void {
+    // Skip this test in 11.2 as we can't save
+    // old revisions anymore like we did.
+    if (version_compare(\Drupal::VERSION, '11.2.0', '>=')) {
+      $this->markTestSkipped();
```

Comment:
It would be nice to put a reference URL with an explanation of what is obsolete. Does it mean that this scenario should not be tested in Drupal 11.2+, or should this test be adapted in the future?